src/GB2260.php

```diff
@@ -15,9 +15,11 @@ public static function getData()
 
     public static function parse($code)
     {
+        //@codeCoverageIgnoreStart
```

Comment:
What's the reason to skip code coverage?

This code block was duplicated in getData() method, too.